.travis.yml

```diff
@@ -17,7 +17,7 @@ dart_task:
 # snapshot if it's available.
 before_script:
   - dart --no-checked --snapshot=bin/pub.dart.snapshot --snapshot-kind=app-jit bin/pub.dart --help
```

Comment:
this might need a `--no-preview-dart-2` for consistency.

Reply:
Done.